### DISCUSSION TITLE
How to show data in graph?

### DISCUSSION THREAD

**Discussion by 420skywalker420** [serenity-is/Serenity#5720] [Help]
Hi guys,

I have dropdown menu with 2 possible cases. When I choose 1 of 2 possible options I have to take data from selected table and show that in graph. My question is how to take data from table and show that in graph?

Thank you!

**Comment by edwardch** [comment]
Take a look at this issue: #2342 for inspiration.

Also look at [Chart.js](https://www.chartjs.org/), which has recently hit v3.

**Reply by 420skywalker420** [reply]
Thank you, sir. I'll try to find solution!
